IntegrationTests/IntegrationScenarios/Scenarios/WebView/WebViewScenarioTest.swift

```diff
@@ -16,13 +16,16 @@ class WebViewScenarioTest: IntegrationTests, RUMCommonAsserts {
         let rumServerSession = server.obtainUniqueRecordingSession()
         // Server session recording Logs send to `HTTPServerMock`.
         let loggingServerSession = server.obtainUniqueRecordingSession()
+        // Server session recording Replays send to `HTTPServerMock`.
+        let srServerSession = server.obtainUniqueRecordingSession()
```

Comment:
question/ Does the web page we display in WebView as part of this test scenario support browser SR? I wonder how this work, it looks great 👍, easy but great coverage.

Reply:
Not yet, but I have enabled native SR in this scenario so that the native RUM views have `has_replay` true. It is the condition to inject the `container` field in Browser RUM events.